notNeededPackages.json

```diff
@@ -1685,6 +1685,14 @@
             "libraryName": "es-module-lexer",
             "asOfVersion": "0.4.1"
         },
+        "es-to-primitive": {
+            "libraryName": "es-to-primitive",
+            "asOfVersion": "v1.3.0"
```

Comment:
I don't think these `v` prefixes are legal, and I am surprised CI did not complain (well, not surprised).

Note the syntax of all of the other ones.

Reply:
Sure, i'd expect the script to have semver.normalize'd them all :-) will fix